frontend/src/app/store/reducers/hydration.reducer.ts

```diff
@@ -0,0 +1,23 @@
+import { ActionReducer, INIT, UPDATE } from '@ngrx/store';
+
+import { AuthState } from '@store/states/auth-state';
+
+export const hydrationMetaReducer = (
```

Comment:
You have reducer that restores your state. Shouldn't you have a meta reducer that would clear your state and local storage on logout?

Reply:
Fixed